Framework/Crystal/src/SetSpecialCoordinates.cpp

```diff
@@ -3,10 +3,15 @@
 #include "MantidAPI/IMDEventWorkspace.h"
```

Comment:
The algorithm is still being used for Peaks workspaces, hence I am not sure that it can be depcrecated.